### ISSUE TITLE
HDMI-CEC not enabled on TV after start (cubox-i, samsung TV)

### ISSUE THREAD

**Issue by martinbalint** [OpenELEC/OpenELEC.tv#3639]
Hi, when OpenELEC 5.0 beta 3 starts, it doesn't respond to TV remote calls. I have to manualy select Kodi device from TV Anynet+ menu. After that, it works great, until next restart.

With same cable,Openelec 4 and Rpi, OpenELEC responds immediately after Kodi start.

Log: http://pastebin.com/aZ2Jg6P5

Doesn't work in: 4.95.4, 4.95.5

**Comment by tunf** [comment]
Still the same in final openelec 5

**Comment by lucaswalther** [comment]
I've been running OpenELEC on a Raspberry Pi with Samsung TV for a couple of years (started w/ version 3.x), and HDMI CEC has always worked perfectly. I didn't even notice any issues after upgrading to 5.0 a little over a week ago (I don't experiment with any of the betas, only stable releases).

However, I just purchased a CuBox i4Pro and set it up with OpenELEC 5.0 last night, and I'm not experiencing the same types of issues described by others here... the HDMI CEC (Samsung calls it "Anynet+") doesn't work initially when the device is first powered on, and also seems to be "lost" after some time (anywhere from a few minutes to several hours). Power cycling the the television, -OR- reconnecting through the television menu usually gets it working again, but there is still a delay of several seconds before the functionality returns.

**Comment by zilexa** [comment]
For me, OpenElec 5.0 with Cubox-i 4pro with Sony W706 tv CEC works perfect.

**Comment by jenkins101** [comment]
works here too

**Comment by c-o-m-m-a-n-d-e-r** [comment]
same problem here with 5.0.8 stable  ...and a philips media suite .... it starts but after a few seconds i have to select hdmi port again, then it works

**Comment by Colargol59** [comment]
Same problem... this issue should be reopened! OpenElec 5.95.1, Samsung LE32C550

**Comment by fritsch** [comment]
Won't help, cause no one is working on it. But here, you are - reopened.

**Comment by tunf** [comment]
@fritsch, thanks or reopening. I think this is the general cec issue with all versions and devices. I have it reported several times. it is pitty that i'm not a programmer, otherwise i would have offered my help.
Is this still due to the libcec issue?

**Comment by fritsch** [comment]
No idea, i don't have a CEC capable device here. CEC3 will soon be integrated - then let's revisit.

**Comment by nebuchar** [comment]
I have the same problem, it works initially but stops working after some seconds

Didn't had this problem before, even with 5.0.8

**Comment by tunf** [comment]
on 6.0.1 still no stable / working cec possible.
After the update and reboots the samsung tv sees cec, connects disconnects, manual selecting cec device there one named kodi. If selected for 1 to 2 seconds able to use remote. Then a disconnect. Looking for cec devices manually again it now shows recorder, select this one, cec working for 1 to 2 seconds then disconnects, manually no device visable that is related to openelec/ kodi. Able to use remote. Starting a plugin/ video or so, cec disconnects (or after about 6 seconds)

**Comment by unclejamal3000** [comment]
I'm also having HDMI CEC problems with my Panasonic TX-49CXW754 and a Cubox-i on all Openelec 6.0.x versions (including the latest 6.0.3). It was a bit more stable with the 6 beta versions (5.95.x) but currently it's unuseable and I switched it off completely.

Problems are:
- TV switches on when Kodi starts although I configured to not wake any devices
- TV swichtes to other HDMI channel (away from Kodi) whenever I start a movie.
- Remote control is not working

If I can help on this issue with logs, tests whatever - let me know.
